src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java

```diff
@@ -1,9 +1,11 @@
 package uk.gov.digital.ho.hocs.workflow.api.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
```

Comment:
@agraham19 I think these are bits of my branch that have made it in here. I've taken them out on the branch we're merging too. `@ToString` is great when debugging, there's always the risk that something will appear in logs that shouldn't and our logging doesn't really require objects to be logged out in full. That being said, if you're not familiar it, I'd recommend learning it as it is easily configured to remove the things you don't want to log.

Reply:
I have addressed this.